docs/linting/Troubleshooting.mdx

```diff
@@ -249,3 +249,9 @@ Rules such as [`no-unsafe-argument`](https://typescript-eslint.io/rules/no-unsaf
 ## My linting feels really slow
 
 If you think you're having issues with performance, see our [Performance Troubleshooting documentation](./troubleshooting/Performance.md).
+
+## Are TypeScript project references supported?
+
+No, TypeScript project references is not yet supported. 
```

Comment:
```suggestion
No, TypeScript project references is not yet supported.
```
I'm currently away from my computer, but I think this might be the problem with the test failing? When I get to my laptop is there a specific command I can run to check? I tried looking in the package.json but none worked. I could be doing it wrong though. 🤔